python-pro-data-1/instalace/instalace-python-vscode.md

```diff
@@ -2,5 +2,4 @@
 
 Pokud zatím nemáš nainstalovaný Python a Visual Studio Code, postupuj dle instrukcí z úvodního kurzu, konkrétně projdi tyto kapitoly:
 * [Instalace jazyka Python](https://stage.kodim.cz/kurzy/uvod-do-progr-1/priprava/jazyky-nastroje/instalace-python)
-* [Instalace editoru Visual Studio Code](https://stage.kodim.cz/kurzy/uvod-do-progr-1/priprava/jazyky-nastroje/instalace-vscode)
 * [Rozšíření VS Code pro Python](https://stage.kodim.cz/kurzy/uvod-do-progr-1/priprava/jazyky-nastroje/instalace-rozsireni-vscode)
```

Comment:
Nejsou všechny odkazy nefunkční, protože odkazují na stage.kodim.cz?

Reply:
Je tam redirect, ale ja to jeste upravim, aby to vubec neukazovalo na _stage_.

Reply:
Jo jasně! Díky